packages/palette/src/elements/RadioGroup/RadioGroup.tsx

```diff
@@ -90,9 +93,21 @@ export class RadioGroup extends React.Component<
   }
 
   render() {
-    const { disabled, onSelect, defaultValue, children, ...others } = this.props
+    const {
+      disabled,
+      disabledText,
+      onSelect,
+      defaultValue,
+      children,
+      ...others
+    } = this.props
     return (
       <Flex flexDirection="column" {...others}>
+        {disabled && disabledText !== undefined && (
```

Comment:
Don't need to explicitly check for undefined here. Also slight fix for form (for instances when it is needed):

```
typeof variableName !== "undefined"
```

When checking for undefined